### PR TITLE
Update css from design team feedback

### PR DESCRIPTION
**Pull request by WilliamXieCrypto**
Update css from design team.

Here's the final view:

image
image

### CODE REVIEW

**Comment by WilliamXieCrypto** [comment]
New update:

image

image

File: apps/block_scout_web/assets/css/components/_dashboard-banner.scss [hunk 1]
```diff
@@ -47,7 +47,7 @@ $dashboard-banner-chart-axis-font-color: $dashboard-stats-item-value-color !defa
 
 .dashboard-banner-network-graph {
   flex-grow: 1;
-  padding: 15px 0 0 0;
+  //padding: 15px 0 0 0;
```
Comment (ysong42) [inline apps/block_scout_web/assets/css/components/_dashboard-banner.scss]: I think we can just removed the unused css. What do you think?

File: apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex [hunk 2]
```diff
@@ -165,6 +81,29 @@
...
+                <div class="dashboard-banner-network-stats-item dashboard-banner-network-stats-item-6">
+                    <h2 class="dashboard-banner-network-stats-label">
+                        <%= gettext "Market Cap" %>
+                    </h2>
+                    <% total_market_cap = market_cap(@market_cap_calculation, @exchange_rate) %>
+                    <span class="dashboard-banner-network-stats-value" data-selector="market-cap" data-usd-value="<%= total_market_cap %>">
+                        </span>
+                </div>
+              <% end %>
+          <% end %>
```
Comment (ysong42) [inline apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex:112]: I think we need to format this part, using indentation with 2 spaces

File: apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex [hunk 3]
```diff
@@ -165,6 +81,29 @@
...
             </span>
           </div>
+
+            <%= if Map.has_key?(@chart_config, :market) do %>
+              <%= if Enum.member?(@chart_config.market, :price) do %>
+                <div class="dashboard-banner-network-stats-item dashboard-banner-network-stats-item-5">
+                    <h2 class="dashboard-banner-network-stats-label">
+                        <%= gettext "Price" %>
+                    </h2>
+                    <span class="dashboard-banner-network-stats-value" data-selector="exchange-rate" data-wei-value="<%= Explorer.Chain.Wei.from(Decimal.new(1), :ether).value %>" data-usd-exchange-rate="<%= @exchange_rate.usd_value %>">
```
Comment (ysong42) [inline apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex]: For this long line, I think we can split it to multiple lines to improve readability.

For example: 
```
<span 
    class="dashboard-banner-network-stats-value" 
    data-selector="exchange-rate" 
    data-wei-value="<%= Explorer.Chain.Wei.from(Decimal.new(1), :ether).value %>" 
    data-usd-exchange-rate="<%= @exchange_rate.usd_value %>"
></span>
```

The below line 102 has a similar issue.

**Review by ysong42 — CHANGES_REQUESTED** [review]
Awesome job!

Thanks for helping to modify the cronos-blockscout css layout!

I mentioned a few code format issues.

And as discussed, we might need further work to support it on mobile.

File: apps/block_scout_web/assets/css/components/_dashboard-banner.scss (hunk 1 above)
Comment (WilliamXieCrypto) [reply apps/block_scout_web/assets/css/components/_dashboard-banner.scss]: Yes, I agree with you.

File: apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex (hunk 2 above)
Comment (WilliamXieCrypto) [reply apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex:112]: Np~, will update IDE.

File: apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex (hunk 3 above)
Comment (WilliamXieCrypto) [reply apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex]: Yes.

**Comment by WilliamXieCrypto** [comment]
@ysong42 I will update this PR and thank you for your feedback.

**Review by crypto-facs — APPROVED** [review]

**Comment by WilliamXieCrypto** [comment]
@ysong42 @crypto-facs Updated the PR, please review again, thank you so much~

**Review by ysong42 — APPROVED** [review]

**Pull request merged**